packages/core/js-client/src/connection/RelayConnection.ts

```diff
@@ -15,16 +15,16 @@
  */
 import { PeerIdB58 } from '@fluencelabs/interfaces';
 import { pipe } from 'it-pipe';
-import { encode, decode } from 'it-length-prefixed';
+import { decode, encode } from 'it-length-prefixed';
 import type { PeerId } from '@libp2p/interface/peer-id';
 import { createLibp2p, Libp2p } from 'libp2p';
 
 import { noise } from '@chainsafe/libp2p-noise';
 import { yamux } from '@chainsafe/libp2p-yamux';
 import { webSockets } from '@libp2p/websockets';
 import { all } from '@libp2p/websockets/filters';
-import { multiaddr } from '@multiformats/multiaddr';
 import type { Multiaddr } from '@multiformats/multiaddr';
+import { multiaddr } from '@multiformats/multiaddr';
```

Comment:
In modern ts can be replaced with
```suggestion
import { multiaddr, type Multiaddr } from '@multiformats/multiaddr';
```

Reply:
Nice suggestion, thanks!